enable/examples/demo/enable/constraints_demo.py

```diff
@@ -8,6 +8,11 @@
 #
 # Thanks for using Enthought open source!
 
+# check kiwisolver installation
+import importlib.util
+if importlib.util.find_spec('kiwisolver') is None:
+    print("kiwisolver import not failed!!!! Please check it is installed.")
```

Comment:
Please reword this message. "import not failed" doesn't make much sense, and we don't need four exclamation marks.

Reply:
Thanks for pointing out. Just changed